proposals/bulk-memory-operations/Overview.md

```diff
@@ -162,14 +162,15 @@ Filling a memory region can be accomplished with `memory.fill`:
 
 TODO: should we provide `memory.clear` and `table.clear` instead?
 
-The [binary format for the data section](https://webassembly.github.io/spec/binary/modules.html#data-section)
+The [binary format for the data section](https://github.com/WebAssembly/design/blob/master/BinaryEncoding.md#data-section)
```

Comment:
In general I'm a bit wary of referencing `BinaryEncoding.md` since it's not kept up-to-date, and uses slightly different terminology. That said, it seems to be clearer for folks to understand, so I'm OK with this.

Reply:
Oh dear I wasn't aware of that! I'll fix this to have `/core/` in the URL and switch back to the original link. I should probably just get more used to the official spec text style :)